knowledge_engine/tests/test_conversion/test_converter.py

```
@@ -51,21 +91,111 @@ def test_convert_document_success_with_dot_extension():
+        "html",
+        config,
+    )
+
+    assert b"Title" in result.markdown_bytes
+    assert b"Body text" in result.markdown_bytes
+
+
+def test_convert_document_local_xml():
+    config = MinerUConfig(api_base_url="http://mineru:8367")
+    xml_bytes = ElementTree.tostring(
+        ElementTree.fromstring("<root><item>Value</item></root>")
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)